x-pack/legacy/plugins/lens/public/indexpattern_plugin/indexpattern.tsx

```diff
@@ -37,6 +37,14 @@ export interface IndexPattern {
   fields: IndexPatternField[];
   title: string;
   timeFieldName?: string | null;
+  hasExistence?: boolean;
```

Comment:
Like we discussed offline, I think we're going to want a custom response from the server that contains the existence information and other processing we want. If that's the case then we'll be able to delete this `hasExistence` name completely because we will always have it. And on a field level we will also always know whether the field exists. So I think these naming questions can be ignored for now, although I've added a TODO here